home.hbs

```diff
@@ -37,16 +37,16 @@
 
         <div class="hero2">
             <div  class="text-heading home-text-heading">
-            <img  class="hero-logo" src="{{asset "images/logo.png"}}">
+            <img  class="hero-logo" src="{{asset "images/logo.png"}}" alt="Winnipeg Police Cause Harm logo.">
```

Comment:
This probably doesn’t matter but maybe the punctuation is superfluous in this and other alt/label attributes?